### ISSUE TITLE
Backup directly to Google Drive

### ISSUE THREAD

**Issue by cstamas** [jabstone/JABtalk#3]
Would be great if it would be possible to perform backup to Google drive directly.

Use case: my kid uses two tablets. One is used by him in school (driven by speech tutor in school), while the other is used at home. Both tablets are set up using his own google ID. Would be good, if we and the teacher could backup our JABTalk settings via drive, to basically _share_ words, images, sounds we have.

Note: while currently it is possible to do this, for non-technical persons (like the teacher is), it is a bit cumbersome: backup, upload it using Google drive on tablet, then download it to device via Drive, and restore.

**Comment by jabstone** [comment]
I agree backing up to Google drive directly would be cool. I suspect (although I could be wrong) the more common use case is when JABtalk is backed up on a different account than the one you wish to restore to. In other words, backup happens on Google account A and you want to restore to Google account B. Not sure how to make that user experience simple. I'm open to ideas though.

**Comment by cstamas** [comment]
While am in Java land, am really 0% in android. Maybe time to learn something new? :)

Re account A vs account B: maybe share between A and B it and make the backup location standard/configurable? Really unsure is drive API all about paths, or also about "origins", ie. "my drive" vs "shared with me"...

**Comment by af1nger** [comment]
Hi, 

We used jab talk on a Samsung phone... The only app installed is jabtalk. Recently, it got destroyed when it got wet. We did back it up regularly but the most recent backup was on phone, not copied to Google drive...we had to start from an old back up and add words as best we could remember... We are being far more careful to copy the back ups to Google drive on new Samsung phone.

Would be great for this reason if backups were saved to Google drive by default/as well... Even a scheduled auto backup would be amazing... Like every week, it saved a copy of itself.

We also have the same Google account signed in on a tablet... We would make edits and changed on the tablet, then back it up and restore that back up on the phone for our kid... This way he is not without his jabtalk phone at all

Thank you by the way for making this app... It has been life changing!
